storybook/storybook-react/src/OrderedList/OrderedList.stories.tsx

```diff
@@ -7,19 +7,29 @@ import { Heading, OrderedList, Paragraph } from '@amsterdam/design-system-react'
 import type { Meta, StoryObj } from '@storybook/react'
 import { exampleOrderedList } from '../shared/exampleContent'
 
-const orderedList = exampleOrderedList().map((text, index) => <OrderedList.Item key={index}>{text}</OrderedList.Item>)
+const orderedListItems = exampleOrderedList().map((text, index) => (
+  <OrderedList.Item key={index}>{text}</OrderedList.Item>
+))
 
 const meta = {
   title: 'Text/Ordered List',
   component: OrderedList,
   args: {
-    children: orderedList,
-    inverseColor: undefined,
+    children: orderedListItems,
+    inverseColor: false,
     markers: undefined,
   },
   argTypes: {
+    inverseColor: { control: 'boolean' },
     markers: { control: 'boolean' },
```

Comment:
We generally hide `children` from the controls

Reply:
Yes, I’ve configured that globally now. It seems that the arg still appears in the table if it has an initial value, e.g. for `Button` or `Heading`.

Reply:
That works. I think we sometimes do want to show the `children` control though, it's useful to be able to change the text of a Button or Heading imo. Is that possible?

Reply:
Whoah! I had checked that and have seen `children` args with this change in effect, but that must have been cache or something – they’re now gone indeed. I’ve specified them explicitly where we want to see them.